webpack/src/module.js

```diff
@@ -13,9 +13,9 @@ var Module = {
       .sort(function(a, b) { return a.length - b.length; });
     var targetFileIndex = targetFile + '/index';
 
-    var asIs = keys.find(function(name) { return name.indexOf(fileName) >= 0; });
-    var asFile = keys.find(function(name) { return name.indexOf(targetFile) >= 0; });
-    var asIndex = keys.find(function(name) { return name.indexOf(targetFileIndex) >= 0; });
+    var asIs = keys.filter(function(name) { return name.indexOf(fileName) >= 0; }).shift();
```

Comment:
Probably it worth in future to convert this to a single cycle, for performance sake.

Reply:
Yes, maybe create a module like utils.js or something.